locales/sk/json.json

```diff
@@ -203,48 +203,48 @@
     "Czech Republic": "Česká republika",
     "Côte d'Ivoire": "Pobrežie Slonoviny",
     "Dark": "Tmavý",
-    "Dashboard": "Panel",
+    "Dashboard": "Prehľad",
     "December": "December",
-    "Decrease": "Znížiť",
+    "Decrease": "Zníženie",
     "Default": "Predvolené",
     "Delete": "Vymazať",
     "Delete Account": "Odstrániť Účet",
     "Delete all notifications": "Odstráňte všetky upozornenia",
-    "Delete API Token": "Odstrániť Token API",
-    "Delete File": "Odstrániť Súbor",
-    "Delete Resource": "Odstrániť Prostriedok",
-    "Delete Selected": "Odstrániť Vybrané",
+    "Delete API Token": "Odstrániť API Token",
+    "Delete File": "Odstrániť súbor",
+    "Delete Resource": "Odstrániť položku",
+    "Delete Selected": "Odstrániť zvolené",
     "Delete Team": "Odstrániť Tím",
     "Denmark": "Dánsko",
-    "Detach": "Odpojiť",
-    "Detach Resource": "Odpojiť Zdroj",
-    "Detach Selected": "Odpojiť Vybrané",
-    "Details": "Údaje",
-    "Disable": "Vypnúť",
+    "Detach": "Odobrať",
+    "Detach Resource": "Odobrať položku",
+    "Detach Selected": "Odobrať zvolené",
+    "Details": "Detaily",
+    "Disable": "Zakázať",
     "Djibouti": "Džibutsko",
-    "Do you really want to leave? You have unsaved changes.": "Naozaj chceš odísť? Máte neuložené zmeny.",
-    "Dominica": "Nedeľa",
+    "Do you really want to leave? You have unsaved changes.": "Naozaj chcete odísť? Chcete zahodiť zmeny.",
+    "Dominica": "Dominica",
     "Dominican Republic": "Dominikánska Republika",
-    "Done.": "Robiť.",
+    "Done.": "Hotovo.",
     "Download": "Stiahnuť",
     "Download Receipt": "Stiahnite si potvrdenie",
-    "Drop file or click to choose": "Presuňte súbor alebo kliknutím vyberte",
-    "Drop files or click to choose": "Presuňte súbory alebo kliknutím vyberte",
+    "Drop file or click to choose": "Presuňte súbor alebo ho vyberte kliknutím",
+    "Drop files or click to choose": "Presuňte súbory alebo ich vyberte kliknutím",
     "Ecuador": "Ekvádor",
     "Edit": "Upraviť",
     "Edit :resource": "Upraviť :resource",
-    "Edit Attached": "Upraviť Pripojený",
+    "Edit Attached": "Upraviť položku",
     "Edit Profile": "Upraviť profil",
     "Editor": "Editor",
-    "Editor users have the ability to read, create, and update.": "Používatelia editora majú možnosť čítať, vytvárať a aktualizovať.",
+    "Editor users have the ability to read, create, and update.": "Používatelia s rolou editor majú možnosť čítať, vytvárať a upravovať položky.",
     "Egypt": "Egypt",
     "El Salvador": "Salvador",
-    "Email": "Mail",
-    "email": "mail",
-    "Email Address": "adresa",
+    "Email": "Email",
```

Comment:
The name also needs to be added to the `_excludes.json` file, otherwise the status will show up as requiring translation.